config_template.cfg

```diff
@@ -913,6 +913,23 @@ CATALYTIC_WALL= ( NONE )
 WALL_ROUGHNESS = (wall1, ks1, wall2, ks2)
 %WALL_ROUGHNESS = (wall1, ks1, wall2, 0.0) %is also allowed
 
+% ------------------------ WALL FUNCTION DEFINITION --------------------------%
+%
+% The von Karman constant, the constant below is only used in the standard wall function model
```

Comment:
I don't think we should do that at this location.
The reason I have not implemtented one von Karman constant to rule them all is because some turbulence models use the von Karman constant (SA for instance), but the default for the specific turbulence model might be different for the different models (and they are never changed). If we change the von Karman constant in the SA model as well,then the other SA constant should also be allowed to change.